python/lsst/analysis/tools/actions/scalar/scalarActions.py

```diff
@@ -53,6 +54,8 @@
 from ...interfaces import KeyedData, KeyedDataSchema, Scalar, ScalarAction, Vector
 from ...math import nanMax, nanMean, nanMedian, nanMin, nanSigmaMad, nanStd
 
+_LOG = logging.getLogger(__name__)
+
```

Comment:
In python I think it's more customary to use `log` instead of `_LOG`, for example https://developer.lsst.io/stack/logging.html#logger-names .